src/css/styles.scss

```diff
@@ -45,45 +43,48 @@ canvas {
   .date {
     background-color: $text-color;
     border-radius: 3px;
```

Comment:
We have a couple instances of border at 3px - might be a good case for a variable!